src/flows/components/header/VersionBullet.scss

```diff
@@ -0,0 +1,41 @@
+.publish-version-bullet--button {
+  margin-right: 40px;
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.publish-version-bullet--button::before {
+  content: '';
+  height: 25px;
+  width: 4px;
+  transform: translate(-480%, 0%) rotate(-90deg);
+
+  @media screen and (max-width: $cf-grid--breakpoint-md) {
+    transform: translate(-380%, 0%) rotate(-90deg);
```

Comment:
the media is here because the component automatically adjusts sizes at the breakpoint.